src/support/libsupportinit.c

```diff
@@ -51,6 +52,31 @@ void libsupport_init(void)
     }
 }
 
+#ifdef __POSIX__
```

Comment:
```suggestion
#if defined(_POSIX_C_SOURCE) && _POSIX_C_SOURCE >= 200112L
```
Julia does not have the `__POSIX__` define to identify a POSIX system (it is non-standard and was used by the Node.js system because they defined it).

Reply:
can i include ```#ifdef __POSIX__ ``` after I have defined it here ```#if defined(_POSIX_C_SOURCE) && _POSIX_C_SOURCE >= 200112L```?